### PR TITLE
aws: aws_db_parameter_group + aws CONTRIBUTING.md doc

### PR DESCRIPTION
**Pull request by talset**
Add

  * aws_db_parameter_group resource
  * aws_iam_access_key
  * aws_cloudwatch_metric_alarm
  * aws_autoscaling_policy
  * aws_iam_user_ssh_key

Update the CONTRIBUTING doc with a "howto" on adding new aws resource

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -90,6 +90,101 @@ Now, you can write your `computeInstance` function. Check-out the other function
...
+		Service: "rds",
+		Documentation: `
+		// GetDBParameterGroups returns all DB parameterGroups based on the input given.
+		// Returned values are commented in the interface doc comment block.
+		`,
+	},
+
+```
+
+Functions are used to generate Get methode in `aws/reader/reader.go`. With our example we should find `GetDBParameterGroups` at th eend when `make generate` will be executed
```
Comment (sdurrheimer) [inline CONTRIBUTING.md]:
```suggestion
Functions are used to generate Get methods in `aws/reader/reader.go`. After a `make generate` execution, we should find a `GetDBParameterGroups` method at the end of the file corresponding to our previous example.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -90,6 +90,101 @@ Now, you can write your `computeInstance` function. Check-out the other function
...
+
+resources = map[ResourceType]rtFn{
+  DBParameterGroup:   dbParameterGroups,
+
+```
+
+const is used to generate resourcetype_enumer.go
+
+
+3. Add the associeted function to generate terraform code
```
Comment (sdurrheimer) [inline CONTRIBUTING.md]:
```suggestion
3. Add the associated function to generate terraform codes.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -90,6 +90,101 @@ Now, you can write your `computeInstance` function. Check-out the other function
...
+	},
+
+```
+
+Functions are used to generate Get methode in `aws/reader/reader.go`. With our example we should find `GetDBParameterGroups` at th eend when `make generate` will be executed
+
+
+2. Add your resource type
+
+The naming is based on terraform resource like `aws_db_parameter_group` without `aws` and replace `bla_foo` by `blaFoo`. It should looks like `dbParameterGroup`. For the function by default start by `lowercase` and end by `s`
```
Comment (sdurrheimer) [inline CONTRIBUTING.md]:
```suggestion
The naming is based on terraform resources like `aws_db_parameter_group` but without the `aws` prefix and Snake case like `bla_foo` needs to be replaced by Camel case  `blaFoo`. The function should start with a `lowercase` and end with a `s`.
The end result should looks like `dbParameterGroups`.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -90,6 +90,101 @@ Now, you can write your `computeInstance` function. Check-out the other function
...
+4. Make generate
+
+Last step is to re-generate the following files with enumer and build/install.
+
+  * aws/reader/reader.go
+  * aws/resourcetype_enumer.go
+
+```
+make generate
+make install
```
Comment (xescugc) [inline CONTRIBUTING.md]: Well, `make install` it's not required, I would say more of a `make test` to see if everything compiles.

**Pull request merged**